### ISSUE TITLE
If java code with default values is generated, type mismatch may occur.

### ISSUE THREAD

**Issue by jxnu-liguobin** [kobylynskyi/graphql-java-codegen#291]
**Is your feature request related to a problem? Please describe.**
graphql schema has a field like `interval: Long = 3600000`
will create java code like `private java.lang.Long interval = 3600000;` but `int` can not assign to `Long` in java
there are two ways resolver it.
1. generate code: `private java.lang.Long interval = 3600000L;`
2. generate code: `private long interval = 3600000;` then need add `types.put ("Long", "long")` in customTypesMapping
and can not use `!=` 
```java
    if (interval != null) {//error,  != cannot applied to long and null
            joiner.add("interval: " + GraphQLRequestSerializer.getEntry(interval));
     }
```
At now i use `types.put("Long", "java.lang.Long")`, so will generate a error java code.
And can not append `L` in graphql. At present, in order to modify the minimum code, I am ready to use the first way.
maybe support the first way is easy, i think `graphql-java-codegen` maybe need support  wrap default value.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by kobylynskyi** [comment]
@jxnu-liguobin thanks for reporting this issue

**Comment by kobylynskyi** [comment]
Will be released as part of 3.0.1 by the end of the week
